cli/events-plugin.md

```diff
@@ -0,0 +1,328 @@
+
+On entering any command, you will be taken to the Adobe I/O CLI Login page. After signing in using your ID, you will be redirected to a page stating that you have successfully logged in to the Adobe I/O CLI. 
+
+![](../img/cli-adobe-io-cli-login.png)
+
+You can now proceed to select the organization, project, and workspace using the following steps:
+
+1. Select the same organization in which you created the project in Adobe Developer Console. You can list the organizations and select one using ***aio console org list***. 
+
+    ![](../img/cli-org-list.png)
+
+    To select an organization, use ***aio console org select <ims_org_id>***.
+
+    ![](../img/cli-org-select.png)
+
+
+2. Select the project you created in Console. You can list the projects using ***aio console project list***.
+
+    ![](../img/cli-project-list.png)
+
+    To select a project, use ***aio console project select <project_id>***.
+
+    ![](../img/cli-project-select.png)
+
+3. Select the workspace from the project you created in Console. By default a Production workspace is created when you create a new project. You can list the workspaces using ***aio console workspace list***.
+
+    ![](../img/cli-workspace-list.png)
+
+    To select a workspace, use ***aio console workspace select <workspace_id>***.
+
+    ![](../img/cli-workspace-select.png)
+
+## Installing and using the plugin
+
+Use the following syntax to install the Events Plugin for the Adobe I/O CLI and issue commands.
+
+```shell
+$ npm install -g @adobe/aio-cli-plugin-events
+$ aio COMMAND
+running command...
+$ aio (-v|--version|version)
+@adobe/aio-cli-plugin-events/1.0.0 darwin-x64 node-v10.18.1
+$ aio [COMMAND] --help
+USAGE
+  $ aio COMMAND
+...
+```
+
+## Commands
+
+There are a number of commands available when using the Events Plugin, please select an area from the following list to learn more: 
+
+* [Common command](#common-command)
+* [Registration commands](#registration-commands)
+* [Provider commands](#provider-commands)
+* [Event Metadata commands](#event-metadata-commands)
+
+### Common command
+
+```shell
+aio event
+```
+
+### Registration commands
+
+* [Manage your Adobe I/O Events Registrations](#manage-your-adobe-io-events-registrations)
+* [Get an Event Registration in your workspace](#get-an-event-registration-in-your-workspace)
+* [List Event Registrations in your workspace](#list-event-registrations-in-your-workspace)
+* [Create an Event Registration](#create-an-event-registration)
+
+#### Manage your Adobe I/O Events Registrations
+
+```shell
+USAGE
+$ aio event:registration
+ 
+OPTIONS
+-v, --verbose Verbose output
+--help Show help
+--version Show version
+```
+
+#### Get an Event Registration in your workspace
+
+```shell
+USAGE
+  $ ./bin/run event:registration:get REGISTRATIONID
```

Comment:
we shoudl avoid ./bin/run everywehre. it should be aio

Reply:
Also, I told you that id get back to you on this. We can also do aio event registration get without `:` and this applies for all commands.